plugins/databases/snowflake/src/main/java/org/apache/hop/core/database/SnowflakeDatabaseMeta.java

```diff
@@ -0,0 +1,401 @@
+/*! ******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2002-2017 by Hitachi Vantara : http://www.pentaho.com
+ *
+ *******************************************************************************
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+
+package org.apache.hop.core.database;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang.Validate;
+import org.apache.hop.core.Const;
+import org.apache.hop.core.gui.plugin.GuiElementType;
+import org.apache.hop.core.gui.plugin.GuiPlugin;
+import org.apache.hop.core.gui.plugin.GuiWidgetElement;
+import org.apache.hop.core.plugins.DatabaseMetaPlugin;
+import org.apache.hop.core.row.ValueMetaInterface;
+import org.apache.hop.core.util.Utils;
+import org.apache.hop.metastore.persist.MetaStoreAttribute;
+
+/**
+ * Contains Snowflake specific information through static final members
+ *
+ * https://docs.snowflake.net/manuals/sql-reference/info-schema.html
+ *
+ * @author Nicolas Adment
+ * @since 30-janv-2020
```

Comment:
@author and @since are not allowed under the apache license because no one has ownership of the code

Reply:
ok for next time, fixed